modules/capability/depinject.go

```diff
@@ -0,0 +1,51 @@
+package capability
+
+import (
+	"cosmossdk.io/core/appmodule"
+	"cosmossdk.io/depinject"
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	modulev1 "github.com/cosmos/ibc-go/api/capability/module/v1"
+	"github.com/cosmos/ibc-go/modules/capability/keeper"
+)
+
+var _ depinject.OnePerModuleType = AppModule{}
+
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (AppModule) IsOnePerModuleType() {}
+
+func init() {
+	appmodule.Register(
+		&modulev1.Module{},
+		appmodule.Provide(ProvideModule),
+	)
+}
+
+type ModuleInputs struct {
```

Comment:
should add godoc here

Reply:
https://github.com/cosmos/ibc-go/pull/6158/commits/c0793248b8f6cd1447e235c3445c5051237a7011